systems/plants/@RigidBodyManipulator/addRobotFromSDF.m

```diff
@@ -312,6 +328,9 @@
         if (options.visual || options.visual_geometry)
          geometry = RigidBodyGeometry.parseSDFNode(geomnode,xyz,rpy,model,body.robotnum,options);
          if ~isempty(geometry)
+           if isa(geometry,'RigidBodyMesh')
+             geometry.scale=.0254; % gazebo dae meshes appear to be in inches
+           end
```

Comment:
We could easily add a tag (manually) with a scale of 0.0254, but the question is whether all of the meshes provided by gazebo/drcsim will have those units by default.